src/notion.ts

```diff
@@ -54,11 +56,18 @@ const updateCard: (
         page_id: pageId,
         properties: {
           [attempt.key]: notionTypeToPropValue(attempt.type, value)
-        } as never
-      })
+        }
+      } as Parameters<typeof notion.pages.update>[0])
       core.info(
         `${attempt.key} was successfully updated to ${value} on page ${pageId}`
       )
+      if (githubUrl && isPR) {
```

Comment:
I'd be tempted to wrap this in it's own try-catch so we can have a more useful error message on which part of this process failed. This update will fail if the card doesn't have the `GitHubLink`

Reply:
Ah I thought it would create that property if missing...that's a bummer. Maybe we can do an action which creates the property if it doesn't exist? Other options would be to create a comment or just add the link as a newline in the card, but couldn't find how to do any of those two on a quick look.